lms/sql_tasks/tasks/fdw/create_from_scratch/00_create.jinja2.sql

```diff
@@ -1,15 +1,31 @@
-DROP SERVER IF EXISTS "{{h_fdw_server_name}}" CASCADE;
-CREATE SERVER {{h_fdw_server_name}} FOREIGN DATA WRAPPER postgres_fdw 
-    OPTIONS (host '{{h_fdw_host}}', port '{{h_fdw_port}}', dbname '{{h_fdw_dbname}}');
+-- In production this must have been either 
+-- provisioned while creating the RDS db or run with and admin user
+-- before this task can run successfully
+CREATE EXTENSION IF NOT EXISTS postgres_fdw;
 
+DROP SERVER IF EXISTS "h_server" CASCADE;
+CREATE SERVER "h_server" FOREIGN DATA WRAPPER postgres_fdw
+    OPTIONS (
+        host '{{h_fdw.host}}', -- SECRET
```

Comment:
You need to sign in to be able to see these logs, but I don't know what happens if you sign in as someone who isn't a Hyp users, so maybe this is best?

It would be good to find out so we don't hide things from ourselves that are handy unnecessarily.